library/asn1write.c

```diff
@@ -232,7 +232,6 @@ int mbedtls_asn1_write_int( unsigned char **p, unsigned char *start, int val )
     int ret;
     size_t len = 0;
 
-    // TODO negative values and values larger than 128
```

Comment:
The code does not work for values larger than 128. Please keep this part of the comment.

Reply:
This is an almost identical change to the original PR #1446 which has already been reviewed and merged. 

This TODO was recorded as issue #1443.

Reply:
I agree with @gilles-peskine-arm: I would prefer to keep this part of the code, if there is a failure around this area and someone has to look at the code, its much better to find a comment explaining the limitations of the implementation rather than having  to read and understand the code in detail or randomly search for issues related to this code...